engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3902,6 +3919,7 @@ private Option add(Option opt)
     final Option originalAppID = add(OptionBuilder.withArgName("application id").hasArg().withDescription("Specify original application identifier for restart.").create("originalAppId"));
     final Option exactMatch = add(new Option("exactMatch", "Only consider applications with exact app name"));
     final Option queue = add(OptionBuilder.withArgName("queue name").hasArg().withDescription("Specify the queue to launch the application").create("queue"));
+    final Option tags = add(OptionBuilder.withArgName("comma separated tags").hasArg().withDescription("Specify the tags for the application").create("tags"));
```

Comment:
I retract my previous statement. There are already files, archives and libjars that take multiple values as comma-separated list. Hadoop also does the same thing. I would say we should be consistent and not make "tag" a special case by doing `-tag a -tag b -tag c`